plugins/PostProcessingPlugin/scripts/DisplayInfoOnLCD.py

```diff
@@ -476,8 +592,63 @@ def message_to_user(self, speed_factor: float):
         if print_start_time != "":
             print_start_str = "Print Start Time................." + str(print_start_time) + "hrs"
         else:
-            print_start_str = "Print Start Time.................Now."
+            print_start_str = "Print Start Time.................Now"
         estimate_str = "Cura Time Estimate.........." + str(print_time)
         adjusted_str = "Adjusted Time Estimate..." + str(time_change)
         finish_str = week_day + " " + str(mo_str) + " " + str(new_time.strftime("%d")) + ", " + str(new_time.strftime("%Y")) + " at " + str(show_hr) + str(new_time.strftime("%M")) + str(show_ampm)
-        return finish_str, estimate_str, adjusted_str, print_start_str
\ No newline at end of file
+
+        # If there are pauses and if countdown is enabled, then add the time-to-pause to the message.
+        if bool(self.getSettingValueByKey("countdown_to_pause")):
+            num = 1
+            for layer in data:
+                for p_cmd in pause_cmd:
+                    if p_cmd in layer or "Do the actual pause" in layer:
+                        adjusted_str += "\n" + self._get_time_to_go(layer.split("TIME_ELAPSED:")[1].split("\n")[0]) + " ET from start to pause #"   + str(num)
+                        num += 1
+        return finish_str, estimate_str, adjusted_str, print_start_str
+
+    def _get_time_to_go(self, time_str: str):
```

Comment:
Could add some description to functions
```suggestion
    def _get_time_to_go(self, time_str: str):
    """
    Converts a time string in seconds to a human-readable format (e.g., "2h30m").

    :param time_str: The time string in seconds.
    :return: A formatted string representing the time.
    """
```

Reply:
Done